schedule/yam/agama_packages_unattended.yaml

```diff
@@ -8,3 +8,4 @@ schedule:
   - installation/grub_test
   - installation/first_boot
   - yam/validate/validate_packages
+  - yam/validate/validate_post_partitioning
```

Comment:
shouldn't go to [schedule/yam/agama_extended_unattended.yaml](https://openqa.suse.de/tests/17444255/settings/schedule/yam/agama_extended_unattended.yaml) ?
so we introduce this extra thing for that test suite.
You will need additional jsonnet for remote arch where there is not yet template generation.

Reply:
Yes, you are right from today`s perspective. The fun fact is that when I looked which YAML to use I looked at this test:
https://openqa.suse.de/tests/17454488#settings 

So that makes me ask, when do we switch to a gitlab based deployment of the Agama job group? If there is no history and everybody adapting it on the fly, we will face such problems more often.

Reply:
I know, I'm waiting too much ... once you push the yaml as it is it is more difficult to do big changes, I'm favor speed and waiting the team pick up with the refactors between feature and feature to develop.
https://progress.opensuse.org/issues/181286
Most likely we could work on this after https://progress.opensuse.org/issues/181295

Reply:
but ok, you are not the first mentioning, I'm going to schedule it :) thanks for the reminder.